manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml

```diff
@@ -299,15 +301,15 @@ spec:
                   - name: OPERATOR_NAME
                     value: "cluster-logging-operator"
                   - name: ELASTICSEARCH_IMAGE
-                    value: "quay.io/openshift/origin-logging-elasticsearch5:latest"
+                    value: registry.svc.ci.openshift.org/ocp/4.4:logging-elasticsearch5
```

Comment:
isn't the old pattern what will get replaced for us by the CI process?

Reply:
Reverting..  this is not supposed to be here